### ISSUE TITLE
SwatchColorPicker A11y Pass

### ISSUE THREAD

**Issue by awalia13** [OfficeDev/office-ui-fabric-react#8489]
Tracking all of the issues brought up with the 3/25 accessibility pass on SwatchColorPicker control
Details on individual bugs can be found on [Sharepoint](https://microsoft-my.sharepoint.com/:x:/r/personal/v-amaver_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B1b786f74-e05c-4c5c-b2db-4e7182cde7b6%7D&action=default&uid=%7B1B786F74-E05C-4C5C-B2DB-4E7182CDE7B6%7D&ListItemId=1902&ListId=%7B395E1ECB-5002-440E-81C5-6A090E98077C%7D&odsp=1&env=prodbubble)

- [ ] Bug1: [Accessibility][Usable]: In scan mode backward navigation, Narrator’s focus and information provided by screen reader is not in sync.

**Comment by dzearing** [comment]
@awalia13 Why?

Could you please indicate whether you see something wrong with the markup or if you feel this is a Narrator issue? It costs a tremendous amount of time for us to investigate these, which we're always willing to do, but it would really help a lot for these reports to be very explicit about what is missing in the component causing the problem, OR, if the screen reader simply isn't behaving as expected, in which these issues should be logged directly to Narrator to save us time.

Could you please dig in here?

**Comment by v-doku** [comment]
@dzearing We have verified this issue on our end with other Screen reader and platform combination(Chrome+NVDA) and it works fine there. Therefore, we have logged this bug as Narrator specific only from our end.

**Comment by dzearing** [comment]
@v-doku Thanks; please do follow up with the Narrator folks to make sure they are tracking the issue. I will close this one.

**Comment by awalia13** [comment]
@dzearing and @micahgodbolt   - As a best practise we used to log the bug at both places (Product database and Narrator database) and mentioned Narrator Bug# in Product Team database and keep product database bug open till the time narrator team will not fix the bug. Once Narrator bug closed we validate the bug logged in product team and then close the Product Team database bug.It will be helpful in easier tracking. Please suggest your thoughts.

**Comment by dzearing** [comment]
@awalia13 I think logging bugs in both places make sense.

If you know that the issue has a fix to be made in the product, please keep it open and indicate in the bug what we're doing wrong.

If you know that the issue is not in the product but rather in the screen reader, you should log it but probably close it immediately, so that devs over here don't spend time investigating to reach the same conclusion. It is expensive to investigate, and then to escalate, and if you're already doing this, then we shouldn't be doing it redundantly.

I think opening the bug (and then immediately closing it) is still helpful for people running into the same issue, who would appreciate the context. Saying something in the bug "this is a known Narrator bug which is being tracked separately" is good, especially helpful if you can provide a link to something publicly accessible and have an ETA on when Narrator will fix.
